tests/integration/v2/distribution/grpc_query_test.go

```
@@ -207,7 +207,7 @@
 
 	var (
 		req    *types.QueryValidatorSlashesRequest
```

golangci-lint (1.62.2)

File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(cosmossdk.io) -s prefix(github.com/cosmos/cosmos-sdk) --custom-order

(gci)